src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,13 +1,12 @@
 <div>
-    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to do
-        this, it needs credentials with access to the projects and repositories you want it to build from.</p>
-
-    <p>To provide Jenkins with credentials you can:</p>
+    <p>When running a job, Jenkins requires credentials to checkout the source code for builds. To do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
```

Comment:
So following on the comment below, I think we should update this as well. It implies the credentials only function is for code checkout, and that makes them seem optional if you have SSH credentials- not necessarily the case. I don't think we need to be specific, but it'd be good to say we use them for checkout and other instances where authenticating with bitbucket is necessary